index.bs

```diff
@@ -386,6 +386,7 @@ The user experience of WebRTC-based video conferencing is enhanced using real-ti
 
 Security Considerations {#security}
 ===================================
+This API is to define low-level API for neural network inference hardware acceleration. Therefore all interfaces defined by this specification are only available within a secure context because it might be a high-value target for attackers. See the Secure Contexts specification [[POWERFUL-FEATURES]]
```

Comment:
Thanks @wonsuk73! This is a nice addition to the Security Considerations section. Here's a slightly altered wording that mentions a concrete example (MITM attack) and gives some more rationale why this is important.

WDYT?

```suggestion
This specification defines a low-level API for neural network inference hardware acceleration. This API is considered a powerful feature [[POWERFUL-FEATURES]] because it grants low-level access to a user's computer. To meet the authentication and confidentiality expectations of a powerful feature and to prevent man-in-the-middle attacks, all interfaces defined by this specification are only available in a secure context.
```